src/quo2/components/wallet/network_routing/view.cljs

```diff
@@ -0,0 +1,197 @@
+(ns quo2.components.wallet.network-routing.view
+  (:require
+    [oops.core :as oops]
+    [quo2.components.wallet.network-routing.animation :as animation]
+    [quo2.components.wallet.network-routing.style :as style]
+    [quo2.theme :as quo.theme]
+    [react-native.core :as rn]
+    [react-native.gesture :as gesture]
+    [react-native.reanimated :as reanimated]
+    [reagent.core :as reagent]
+    [utils.number]))
+
+(defn- slider
+  [slider-shared-values]
+  [rn/view {:style style/slider-container}
+   [reanimated/view {:style (style/slider slider-shared-values)}]])
+
+(defn network-bar
+  [_]
+  (let [detecting-gesture?      (reagent/atom false)
+        amount-on-gesture-start (atom 0)]
+    (fn [{:keys                [total-width total-amount on-press on-new-amount allow-press?]
+          {:keys [amount-shared-value
+                  max-amount]} :bar
+          :as                  props}]
+      (let [slider-width-shared-value   (reanimated/use-shared-value 4)
+            slider-height-shared-value  (reanimated/use-shared-value 32)
+            slider-opacity-shared-value (reanimated/use-shared-value 0)
+            network-bar-shared-value    (reanimated/interpolate amount-shared-value
+                                                                [0 total-amount]
+                                                                [0 total-width])
+            width->amount               #(/ (* % total-amount) total-width)]
```

Comment:
perhaps a named param is clearer here, rather than `%` ( which I assume is width) 🤔

Reply:
oh yes, since the name of the function is `width->amount` I think is clear enough that the parameter received is `width`.

I found a little redundant:
`width->amount (fn [width] (/ ( ,,,)))`

Also naming functions using `->` is pretty common in Clojure, so I think it's clear enough. :)